app/views/layouts/_filters.html.haml

```diff
@@ -0,0 +1,24 @@
+/ Filters for ada, unisex and map toggle
+.row
+  .col-xs-12.headroom
+    #filters.clearfix
+      .btn-group{:data => {:toggle => "buttons"}}
+        - if params[:ada].present? && params[:ada] == 'true'
```

Comment:
I know there's a way to combine the indented lines in haml (or maybe not?), but I got pretty sick of looking up haml docs.